src/PageRouter.tsx

```diff
@@ -26,8 +28,8 @@ const PageRouter: React.FunctionComponent<PageRouterProps> = props => {
     });
   }, []);
   if (loading) {
-    const result = getRedirectResult(auth);
-    return <div>Loading...</div>;
+    // const result = getRedirectResult(auth);
```

Comment:
I haven't looked at how our auth works. Is this not needed?

Reply:
I don't believe it is, since the value isn't used, but I could be wrong. The documentation suggests that it's just for getting the result of a page redirect w/o side-effects, and the the result variable is unused.

Reply:
Ah okay, as long as this isn't responsible for doing the redirect, then sounds good